### DISCUSSION TITLE
How to have custom text for labels

### DISCUSSION THREAD

**Discussion by rpivovar-godaddy** [observablehq/plot#1044] [Q&A] [answered]
<img width="195" alt="Screen Shot 2022-09-08 at 9 41 15 PM" src="https://user-images.githubusercontent.com/89860869/189254545-e37352e1-4561-49ba-9b54-46aee737e971.png">

I have charts which can have only two values along the y-axis: **0** or **1**. Currently the y-axis labels are these two numbers, but I'd like to instead change the labels to say "Up" and "Down".

Is there a way to do this?

**Comment by mbostock** [comment, marked as answer]
One way to do this is to use the _y_-scale **tickFormat** option:

```js
Plot.plot({
  y: {
    tickFormat: d => d ? "Up" : "Down"
  },
  marks: …
})
```

**Reply by rpivovar-godaddy** [reply]
That worked -- thanks!
